draft-packages/form/KaizenDraft/Form/decision-tokens.scss

```diff
@@ -0,0 +1,75 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/color-vars";
+
+// General purpose tokens
+$dt-color-form-border-color: var(
+  --kz-color-iron-500,
```

Comment:
I want to keep using the raw variables here so I can control the fallback. The other variables should all use `kz-var-*`